lib/commons/aria/attributes.js

```diff
@@ -37,13 +37,9 @@ aria.allowedAttr = function(role) {
  * @memberof axe.commons.aria
  * @instance
  * @param  {String} att The attribute name
- * @param  {Object} options Use `flagUnsupported: true` to report unsupported attributes
  * @return {Boolean}
  */
-aria.validateAttr = function(att, { flagUnsupported = false } = {}) {
+aria.validateAttr = function(att) {
```

Comment:
I know you are only removing an argument here, could you name the function here, as it will be easier to debug in the long run.

```
aria.validateAttr = function validateAttr(att)....
```